examples/snippets/modules/ROOT/examples/base-images-override/build.sbt

```diff
@@ -3,8 +3,9 @@ import sbt.Keys._
 
     //tag::docs-projectSetup-example[]
 lazy val sampleApp = (project in file("."))
+    .enablePlugins(CloudflowApplicationPlugin)
     .settings(
-      cloudflowAkkaBaseImage := "myRepositoryUrl/myRepositoryPath:2.0.10-cloudflow-akka-2.6.6-scala-2.12",
+      cloudflowDockerBaseImage := "myRepositoryUrl/myRepositoryPath:2.0.10-cloudflow-akka-2.6.6-scala-2.12",
```

Comment:
I'm not sure if there are other places in docs where we should recommend the new sbt setting instead of the deprecated ones.